guides/source/upgrading_ruby_on_rails.md

```diff
@@ -535,7 +535,7 @@ ActiveSupport::JSON::Encoding.time_precision = 0
 Previously, Rails allowed inline callback blocks to use `return` this way:
 
 ```ruby
-class ReadOnlyModel < ActiveRecord::Base
+class ReadOnlyModel < ApplicationRecord
   before_save { return false } # BAD
```

Comment:
Not sure if we should upgrade the `upgrading rails guide` as the the lines are relevant for that particular version. For eg. if I am migrating Rails 4.0 to 4.1, `ApplicationRecord` is not present, so the guide will become invalid.